hpx/components/containers/partitioned_vector/partitioned_vector.hpp

```diff
@@ -732,6 +732,18 @@ namespace hpx
         }
 
     public:
+
+        std::vector< hpx::id_type > get_partitions_ids()
+        {
+            std::vector< hpx::id_type > ids;
+
+            for(auto const part_data : partitions_)
```

Comment:
This is for another functionality extending the concept of array_view. If an array_view needs a data that is not in its locality, make a copy and return it by value.